packages/react/stories/bruk.mdx

```diff
@@ -147,3 +147,21 @@ Dette kan løses ved å f.eks bruke [attribute selector](https://developer.mozil
   …
 </table>
 ```
+
+### Hjelpeklasser
+
+Vi benytter oss av enkelte hjelpeklasser som kan brukes mer generelt på tvers av komponenter.
+
+#### `ds-sr-only`
+Skjuler et element visuelt mens det fortsatt er tilgjengelig for skjermlesere.
+
+#### `ds-readonly-icon`
+Legger til et hengelås-ikon via `::before` til elementet klassen settes på. Brukes blant annet i `Field` på label når input er `readonly`. <span className='ds-readonly-icon'>Eksempel</span>.
```

Comment:
Tror kanskje vi ikke skal dokumentere denne, for den har en del logikk som kjører internt. Kanskje vi skal markere den som intern med `_ds-readonly-icon` ellerno i koden også (eller hva konvensjonen ble)? ☺️

Reply:
Jeg fjernet denne